src/AstroBase.jl

```diff
@@ -1,5 +1,25 @@
 module AstroBase
 
+export tio_locator
+const J2000 = 2451545.0
+const DAYS_PER_CENTURY = 36525.0
+const ARCSEC_TO_RAD = 4.848136811095359935899141e-6
```

Comment:
In `AstroLib.jl` there is a `sec2rad` function (and its inverse `rad2sec`) which preserves precision, perhaps it could be moved here, instead of using a fixed precision constant?

Reply:
I'll move that here.